Sensus.iOS.Tests/AppDelegate.cs.txt

```diff
@@ -0,0 +1,60 @@
+// ***********************************************************************

```

Comment:
Why do we have these .txt files?

Also, I see several Resource.designer.cs files. Why do we have them now that they're covered by .gitignore?